exercises/concept/fundamentals/.docs/instructions.md

```diff
@@ -0,0 +1,66 @@
+# Instructions
+
+Once upon a time, there was a bustling city called Cyberspace.
+Cyberspace was a digital metropolis, with millions of inhabitants logging in and out every day.
+To keep track of all these users, Cyberspace had a special file called /etc/passwd.
+
+The /etc/passwd file was like a phonebook for the city.
+It listed every user who had an account on the system, along with some important information about each user:
+
+| n   | field |
+| --- | --- |
+| 1   | username |
+| 2   | password ("x" indicates the actual password is stored elsewhere, for security) |
+| 3   | UID, a unique numerical user ID |
+| 4   | GID, a unique numerical group iD (typically the same as the UID) |
+| 5   | user info (typically the user's full name, but can include other information) |
+| 6   | home directory |
+| 7   | login shell |
+
+The fields are separated with a colon character.
+This is a typical entry:
+
+```none
+glennj:x:1010:1010:Glenn Jackman:/home/glennj:/usr/local/bin/fish
+```
+
+The file was managed by a team of system administrators, who were responsible for creating new accounts, updating existing accounts, and removing old accounts.
+The system administrators were the gatekeepers of Cyberspace, making sure that only authorized users could access the system.
+
+One day, a new system administrator named Alex was hired to join the team.
+Alex was excited to take on the job, and they quickly learned the ins and outs of the /etc/passwd file.
+They learned how to add new users to the system, how to update user information, and how to remove old users.
+
+As Alex was working on the /etc/passwd file, they noticed something strange.
+Some users had strange or unusual home directories.
+Some users had their home directories set to /dev/null, while others had their home directories set to strange paths that didn't exist.
+
+Alex realized that these users were not real people, but rather fake accounts that had been created by hackers.
+The hackers had found a way to exploit a vulnerability in the system, and had created these fake accounts to gain access to Cyberspace.
+
+Alex knew that they needed to take action.
+They used awk to analyze the /etc/passwd file, and wrote a script to identify all the fake accounts.
```

Comment:
Should /dev/null and awk be in backticks?

Reply:
I forgot about /dev/null, I need to add that to the exercise's sample file.